.github/workflows/build_docker_images.yml

```diff
@@ -132,67 +89,13 @@ jobs:
             type=ref,event=tag,enable=true,format=latest
 
       - name: Write app version info
```

Comment:
This step was not previously happening for the `connector-proxy-demo` image, but with this refactor it is happening, to be consistent with the other two. Let me know if that was intentionally omitted previously!

Reply:
seems great. this sets us up to add this information into an api method in that app, which seems like it would have no downside.